source/CollisionSet.cpp

```diff
@@ -295,6 +301,12 @@ const vector<Collision> &CollisionSet::Line(const Point &from, const Point &to,
 			const Government *iGov = it->body->GetGovernment();
 			if(it->body != target && iGov && pGov && !iGov->IsEnemy(pGov))
 				continue;
+			if(collisionType == CollisionType::SHIP)
+			{
+				Ship *ship = reinterpret_cast<Ship *>(it->body);
+				if(ship != target && ship->CanBeCarried() && ship->IsDisabled())
+					continue;
+			}
```

Comment:
This and a couple of lines above are duplicated, make it a lambda maybe? Or a local function.